### PR TITLE
download the windows sdk directly

### PR DESCRIPTION
**Pull request by devoncarew**
Re-enable the windows bot. Will need to ping this PR a few times.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -3,18 +3,15 @@
 # BSD-style license that can be found in the LICENSE file.
 
 install:
-  # Investigate scoop?
-  - choco install -y dart-sdk -version 1.11.1
+#  - choco install -y dart-sdk -version 1.11.1
+  - ps: wget https://storage.googleapis.com/dart-archive/channels/dev/release/1.12.0-dev.5.1/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
```
Comment (sethladd) [inline appveyor.yml]: would it make sense to download whatever the latest dev channel is?

https://gsdview.appspot.com/dart-archive/channels/be/raw/latest/sdk/dartsdk-windows-x64-release.zip

Comment (devoncarew) [inline appveyor.yml]: Yup, totally. I'm just looking to get a build through once though.

Comment (devoncarew) [inline appveyor.yml]: Thanks for the link :)

**Comment by devoncarew** [comment]
Fix https://github.com/dart-lang/dartdoc/issues/795

@sethladd 

Will squash -

File: appveyor.yml [hunk 2]
```diff
@@ -3,18 +3,16 @@
 # BSD-style license that can be found in the LICENSE file.
 
 install:
-  # Investigate scoop?
-  - choco install -y dart-sdk -version 1.11.1
+# TODO: Switch back to chocolatey once 1.12.0 is available.
+#  - choco install -y dart-sdk -version 1.11.1
+  - ps: wget https://gsdview.appspot.com/dart-archive/channels/be/raw/latest/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
```
Comment (sethladd) [inline appveyor.yml]: are we sure we want 'be' channel? shouldn't we use dev ?

Comment (devoncarew) [inline appveyor.yml]: will change to dev

**Comment by sethladd** [comment]
lgtm

thank you!

**Pull request merged**